lib/pages/sharing_page.dart

```diff
@@ -266,8 +266,7 @@ class SharingPageState extends State<SharingPage> {
                 RichText(
                     text: new TextSpan(children: [
                       new TextSpan(
-                          text:
-                              "Then, share the link to using one of these buttons:",
+                          text: "Share the link to using one of these buttons:",
```

Comment:
Wait why isn't this share the link **by** using one of these buttons

Reply:
Joseph keeps asking for changes but I think we need to say it's a link to add the user.